receiver/githubreceiver/metadata.yaml

```diff
@@ -5,7 +5,7 @@ sem_conv_version: 1.27.0
 status:
   class: receiver
   stability:
-    alpha: [metrics]
+    alpha: [metrics,tracing]
```

Comment:
should this be `traces` according to https://github.com/open-telemetry/opentelemetry-collector/blob/main/cmd/mdatagen/metadata-schema.yaml?

Reply:
correct!